api/apps/api/src/modules/scenarios-features/scenario-features.service.ts

```diff
@@ -79,6 +79,14 @@ export class ScenarioFeaturesService extends AppBaseService<
     ];
   }
 
+  async isFeaturePresentInAnyScenario(featureId: string): Promise<boolean> {
+    const count = await this.remoteScenarioFeatures.query(
+      `select count(*) from scenario_features_data where feature_class_id in (select id from features_data where feature_id = '${featureId}')`,
+    );
```

Comment:
a couple of things:
- I strongly advise against using template strings in queries - even if we pay close attention, it's too easy to accidentally overlook something that includes, somewhere deep up the chain of calls, some user input
```suggestion
    const count = await this.remoteScenarioFeatures.query(
      `select count(*) from scenario_features_data where feature_class_id in (select id from features_data where feature_id = $1)`,
      [ featureId ]
    );
```